### ISSUE TITLE
No error with trailing commas

### ISSUE THREAD

**Issue by Mottie** [stylelint/stylelint#3053]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Trailing commas should show as an error... new rule needed?

> Which rule, if any, is this issue related to?

Not sure, but the `no-descending-specificity` rule was mentioned in https://github.com/stylelint/stylelint/issues/1156#issuecomment-215284836

> What CSS is needed to reproduce this issue?

If the follow css is entered into the [Stylelint demo](https://stylelint.io/demo/)

```css
body, {
  color: red !important;
}
```

> What stylelint configuration is needed to reproduce this issue?

Default rules set in the demo - https://stylelint.io/demo/

> Which version of stylelint are you using?

Newest

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Demo site

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Standard syntax. It's just not valid standard syntax because of a trailing comma.

> What did you expect to happen?

Big red flashing lights. Panic in the streets.

> What actually happened (e.g. what warnings or errors you are getting)?

Nothing

**Comment by hudochenkov** [comment]
Thanks for the report and for using the template.

We had `selector-no-empty` for few releases, but then deprecated it. This is the reason: https://github.com/stylelint/stylelint/issues/2047#issuecomment-261382105.

Also from [changelog](https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md#780):

> -   It is beyond the scope of stylelint's core package to effectively validate against the CSS spec. Please investigate [csstree](https://github.com/csstree/csstree) and [css-values](https://github.com/ben-eb/css-values) for this functionality, and contribute to those projects and to stylelint plugins wrapping them. csstree already has a [stylelint plugin](https://github.com/csstree/stylelint-validator), and css-values needs one to be developed. The following rules are deprecated for this reason.
>     -   `media-feature-no-missing-punctuation`.
>     -   `selector-no-empty`.
